app/bin/run-migrations.sh

```diff
@@ -1,2 +1,12 @@
 #!/bin/sh
-exec npm run migrate
+#
+set -euo nounset
+
+# Run migration with retries
+#
+while !(sleep 5 && npm run migrate); do
+  echo "Waiting 10 seconds for Postgres to start"
+  sleep 5
+done
+
+echo "Migrations complete!  Exiting."
```

Comment:
I'm not sure whether this should be added in - this script is executed in the pre-deploy hook of the deployment configuration. For all intents and purposes, I believe this should be fine. However, I would change the echo message to "Waiting for database migration to complete" as this would more properly reflect what it's attempting to do.

Also, is there any particular reason why this is waiting 5 seconds instead of 10 like in Keycloak (magic number question)?

Reply:
Looks like 2 * 5 = 10 on the sleeps.

Reply:
It's arguably better, since containers can come up out of order.  We could add a retry cap if that's preferable.

Reply:
@jujaga How about "Retrying database migration in 10 seconds"?  The line will only echo when database migration has been unsuccessful and has to retry.